src/Avalonia.Controls/Window.cs

```diff
@@ -704,6 +704,10 @@ private void ShowCore(Window? parent)
             PlatformImpl?.Show(ShowActivated, false);
             Renderer?.Start();
             OnOpened(EventArgs.Empty);
+
+            // Window must manually raise Loaded/Unloaded events as it is a visual root and
+            // does not raise OnAttachedToVisualTreeCore/OnDetachedFromVisualTreeCore events
+            OnLoadedCore();
```

Comment:
Not sure how I feel about this yet. The TopLevel also has OnOpened and OnClosed callbacks. It may be better to do this in the TopLevel itself and actually add an OnOpenedCore() and OnClosedCore() as well. @maxkatz6 any thoughts?

In addition, we are loosing the loaded queue functionality since that is implemented in Control.OnAttachedToVisualTreeCore(). I may move that queuing functionality out to OnLoadedCore() itself. Will think a bit more on that but it may actually be a better implementation and match with OnUnloadedCore() which also now works with the queue directly.

Reply:
> I may move that queuing functionality out to OnLoadedCore() itself

Makes sense.

> It may be better to do this in the TopLevel itself

Maybe. I am actually not sure if TopLevel has enough information to raise loaded/unloaded. As on mobile platforms and wasm there is no window opening, and root is also not attaching to anything. It can be done later in other PR though. For now, I think it's fine to move it to the WindowBase.

Reply:
> For now, I think it's fine to move it to the WindowBase.

Agreed, moved to WindowBase as overrides of the virtual OnOpened/OnClosed for now.

Reply:
>>I may move that queuing functionality out to OnLoadedCore() itself
>
>Makes sense.

I couldn't move everything into OnLoadedCore. The issue is I need private members that really shouldn't be publicly or internally exposed. However, I don't have access to these since the main processing code is running on the dispatcher. So I just created another method that can be called from WindowBase as well.